README.md

```diff
@@ -51,6 +51,7 @@ Later on, you can reference this tutorial and [the official docs](https://derbyj
 
 ### Hello world
 
+<!-- i would place HTML here so that you can open up this link in a new tab https://stackoverflow.com/questions/4425198/can-i-create-links-with-target-blank-in-markdown -->
```

Comment:
I think keeping the links as is is 👌for now b/c people can always cmd-click, and if they do click-through, they can click back (and it should scroll to the right location). Opening in a new-tab by default, though convenient, would break user expectations of how Github links work